tests/config/UNI-1338.cfg

```diff
@@ -0,0 +1,8 @@
+sp_fparen_brace                 = add
+indent_columns                  = 4
+indent_with_tabs                = 0
+indent_class                    = true
+nl_class_leave_one_liners       = true
+nl_end_of_file                  = force
+nl_end_of_file_min              = 1
+sp_inside_braces                = add
```

Comment:
This test (config, input, output) does not seem to belong to this PR. The new option is missing here.